src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface FruitDao {
+    void put(String fruit, Integer quantity);
+
+    void subtract(String fruit, int quantity);
+
+    void addition(String fruit, int quantity);
+
+    Map<String, Integer> getAll();
+
```

Comment:
```suggestion
    Set<Map.Entry<String, Integer>> getEntries();
```
let's not return storage itself, better return it's `entrySet()`